src/Nethermind/Nethermind.Store/MemDb.cs

```diff
@@ -59,6 +58,20 @@ public class MemDb : IDb
             }
         }
 
+        public KeyValuePair<byte[], byte[]>[] this[byte[][] keys]
```

Comment:
this should be an extension method if needed?

Reply:
No, for MultiGet on proper rocks db